src/raftstore/store/store.rs

```diff
@@ -221,14 +221,14 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             region_peers: HashMap::default(),
             merging_regions: Some(vec![]),
             pending_raft_groups: HashSet::default(),
-            split_check_worker: Worker::new("split check worker"),
-            region_worker: Worker::new("snapshot worker"),
-            raftlog_gc_worker: Worker::new("raft gc worker"),
-            compact_worker: Worker::new("compact worker"),
+            split_check_worker: Worker::new("split-check"),
+            region_worker: Worker::new("snapshot-worker"),
+            raftlog_gc_worker: Worker::new("raft-gc-worker"),
+            compact_worker: Worker::new("compact-worker"),
             pd_worker,
-            consistency_check_worker: Worker::new("consistency check worker"),
-            cleanup_sst_worker: Worker::new("cleanup sst worker"),
-            apply_worker: Worker::new("apply worker"),
+            consistency_check_worker: Worker::new("consistency-chk"),
```

Comment:
consistency-checker

Reply:
That's ok, although the name overflows 🤣

Reply:
use full name at first